popups/cloud-games/popup.js

```diff
@@ -92,8 +92,13 @@ export default async ({ addon, msg, safeMsg }) => {
     computed: {
       projectsSorted() {
         return this.projects.sort((b, a) => {
-          if (a.amt !== b.amt) return a.amt - b.amt;
-          return a.timestamp - b.timestamp;
+          if (b.online == true) return -99;
```

Comment:
`===`

Reply:
you don't need === or ==....
remember, that just outputs a boolean that is the exact same as before the equality check

Reply:
@GrahamSH-LLK Our ESLint check will complain if we use `==`

Reply:
@WorldLanguages Graham prob is saying `if (b.online)` is enough

Reply:
I was